testlab1-2.c

```diff
@@ -0,0 +1,304 @@
+#include "testLab.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+static int testN = 0;
+static int seed[] = {1234u, 383u};
+static int LabTimeout;
+
+static const struct {const char *const in; int n; int out[64];} testInOut[] = {
+    {"example\nthis is simple example", 11, { 0, 0, 0, 0, 0, 0, 1, 14, 1, 16, 7}},
+    {"x\n", 1, {0}},
+    {"x\noasdkxmnosaxkmxokasmdx", 9, {0,6, 1, 12, 1, 15, 1, 22, 1}},
+    {"0123456789abcdef\n0123456789abcdef", 18, {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 16}},
+    {"0123\n01230123", 8, {0, 0, 0, 0, 1, 4 ,5 ,4}},
+    {"0123\n0123x0123", 8, {0, 0, 0, 0, 1, 4 ,6 ,4}},
+    {"0123\n012300123", 10, {0, 0, 0, 0, 1, 4 ,5, 1 ,6 ,4}},
+    {"012013\n0123101201343", 10, {0, 0, 0, 1, 2, 0, 1, 3, 6, 6}},
+    {"abcabc\nabcabcabcabcabc", 14, {0, 0, 0, 1, 2, 3, 1, 6, 4, 6, 7, 6, 10, 6}},
+    {"0123456789abcdef\n01234", 16, {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}},
+    {"0101230101234\n0101010122301341010101230101234", 25, {0, 0, 1, 2, 0, 0, 1, 2, 3, 4, 5, 6,  0, 1, 4, 3, 4, 5, 5, 12, 2, 17, 4, 19, 13}},
+    {"\nasdbaf", 0, {0}},
+    {"\xE0\xE1\xE2\xE3\n\xE0\xE1\xE2\xE3\xE0\xE1\xE2\xE3",8,{0, 0, 0, 0, 1, 4, 5, 4}}, // абвг\nабвгабвг
+    {"\xE0\xE1 \xE2\xE3\n\xE0\xE1 \xE2\xE3\xE0\xE1\xE2\xE3", 7, {0, 0, 0, 0, 0, 1, 5}}, // аб вг\nаб вгабвг
+    {"\xE0\xE1 \xE2\xE3\n\xE0\xE1\xE2\xE3\n\xE0\xE1 \xE2\xE3", 9, {0, 0, 0, 0, 0, 1, 2, 6, 5}}, // аб вг\nабвг\nаб вг
+    {"000001\n000000000010", 18,{0, 1, 2, 3, 4, 0, 1, 5, 2, 5, 3, 5, 4, 5, 5, 5, 6, 6}}
+};
+
+static int FeedFromArray(void)
+{
+    FILE *const in = fopen("in.txt", "w+");
+    if (!in) {
+        printf("can't create in.txt. No space on disk?\n");
+        return -1;
+    }
+    fprintf(in, "%s", testInOut[testN].in);
+    fclose(in);
+    return 0;
+}
+
+static int CheckFromArray(void)
+{
+    FILE *const out = fopen("out.txt", "r");
+    int i, passed = 1;
+    if (!out) {
+        printf("can't open out.txt\n");
+        testN++;
```

Comment:
++testN -- globally in this tester

Reply:
other tests use both testN++ and ++testN, but most of all testN++
Also I can see something like [this](https://github.com/Evgueni-Petrov-aka-espetrov/TestDriver/blob/39e2debacd9ab2511fa6b502a759ec31edc0d762/testlab4.c#L103) and [this](https://github.com/Evgueni-Petrov-aka-espetrov/TestDriver/blob/39e2debacd9ab2511fa6b502a759ec31edc0d762/testlab4.c#L116)